code_share/app_code_share/migrations/0001_initial.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
Migrations shouldn't be here. Please add migration folders to `.gitignore` file.